src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,80 @@
+package lotto.view;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class InputView {
+    private static final int MINIMUM_CANDIDATE_NUMBER = 1;
+    private static final int MAXIMUM_CANDIDATE_NUMBER = 45;
+    private static final int LOTTO_NUMBER_COUNT = 6;
+
+    private static Scanner scanner = new Scanner(System.in);
+
+    private InputView(){
+    }
+
+    public static String inputMoney() {
+        System.out.println("구입금액을 입력해 주세요.");
+        String money = scanner.nextLine();
+
+        return money;
```

Comment:
```suggestion
        return scanner.nextLine();
```
요런 친구는 바로 리턴해도 좋겠네요 :)